src/schema/resolvers.js

```diff
@@ -172,6 +172,35 @@ module.exports = {
         cursor.skip(skip);
       }
       return await cursor.toArray();
+    },
+
+    syncInfo: async (root, {}, {mongo: {Blocks}}) => {
+      let options = {
+        "limit": 1,
+        "sort": [["blockNum", 'desc']]
+      }
+      let syncBlockNum = null;
+      let blocks;
+      try {
+        blocks = await Blocks.find({}, options).toArray();
+      } catch(err){
+        console.error(`Error query latest block from db: ${err.message}`);
+      }
+
+      if(blocks.length > 0){
+        syncBlockNum = blocks[0].blockNum;
+      }
+
+      let chainBlockNum = null;
```

Comment:
Define all variables at the beginning of a function.

Reply:
@chrisli30 dunt think it is necessary. No linter style requires it